### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getEntries' of undefined

### ISSUE THREAD

**Issue by gaieges** [tualo/tualo-git-context#9]
[Enter steps to reproduce below:]
1. Start up Atom
2. Enter text

**Atom Version**: 0.186.0
**System**: Mac OS X 10.9.5
**Thrown From**: [tualo-git-context](https://github.com/tualo/tualo-git-context) package, v0.3.0
### Stack Trace

Uncaught TypeError: Cannot read property 'getEntries' of undefined

```
At /Users/me/.atom/packages/tualo-git-context/lib/tualo-git-context-view.coffee:191

TypeError: Cannot read property 'getEntries' of undefined
    at module.exports.TualoGitContextView.refreshTree (/Users/me/.atom/packages/tualo-git-context/lib/tualo-git-context-view.coffee:236:18)
```
### Commands

```
     -0:07.7 core:paste (atom-text-editor.editor)
  2x -0:05.1 editor:newline (atom-text-editor.editor)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  },
  "tualo-git-context": {}
}
```
### Installed Packages

``` coffee
# User
tualo-git-context, v0.3.0

# Dev
No dev packages
```

**Comment by thomashoffmann1979** [comment]
Thanks a lot for filing this issue, i have fixed it in version 0.5.0.

**Comment by gaieges** [comment]
confirmed, thanks!
